### ISSUE TITLE
How to detect AI images? Avoid fake news/topics/tweets/etc...

### ISSUE THREAD

**Issue by RafaBlockDev** [twitter/the-algorithm#677]
### **First Issue** 📝
One issue related to AI images that people share on Twitter is the potential for the spread of fake or manipulated images. With the advancements in AI, it has become easier to create convincing fake images that can be difficult to distinguish from real ones.

### **Second issue** 💀
Another issue is the ethical implications of using AI to generate images. For example, some AI systems have been trained on biased datasets, which can lead to perpetuating stereotypes and discrimination. This can be particularly problematic when it comes to facial recognition technology, where biases in the data can result in misidentification of individuals or groups.

### **Describe alternatives you've considered** 🤓

**_Promoting transparency and accountability:_** To address concerns related to fake or manipulated images, it is important to promote transparency and accountability in AI image generation. This includes developing standards and regulations for AI image generation and ensuring that AI systems are designed and tested to ensure they are transparent, auditable, and accountable.

_**Improving data quality:**_ AI models trained on biased or low-quality data can perpetuate harmful stereotypes and biases. Therefore, it is important to improve the quality and diversity of data used to train AI image generation models. This includes ensuring that data is representative and inclusive of all populations and that any biases are identified and addressed.

_**Ensuring privacy and consent:**_ AI image generation can raise serious concerns about privacy and consent. Therefore, it is important to ensure that AI-generated images are used ethically and with the informed consent of the individuals involved.

_**Investing in research and development:**_ Finally, investing in research and development of AI image generation can help address some of the challenges and concerns related to this technology. This includes developing new algorithms and techniques to improve the quality and accuracy of AI-generated images, as well as exploring new applications of AI image generation in areas such as healthcare and entertainment.

Here is an example of an image generate by AI 😳

<img width="539" alt="Screen Shot 2023-03-31 at 22 45 14" src="https://user-images.githubusercontent.com/86583080/229266115-a7c22f2a-50fe-49bc-bc9c-3dbfb8079875.png">

**Comment by RafaBlockDev** [comment]
### Solution?

**Tensorflow**

**Comment by ValZapod** [comment]
GPT 4 achived AGI level. https://arxiv.org/abs/2303.12712 There is no way to really recognize AI text: https://twitter.com/rlacombe/status/1641113842200592396?t=59oTjrVkmleIqOo51QyvEg&s=19

And finally, even though images have much more data, AGI will be able to obfuscate AI images for all purposes.

**Comment by RafaBlockDev** [comment]
@ValZapod Why do not use models from Tensorflow like: 
1. [Inception](https://www.tensorflow.org/api_docs/python/tf/keras/applications/inception_v3/InceptionV3)
2. [Resnet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet)
3. [MobileNet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet/MobileNet)

**Comment by ValZapod** [comment]
> @ValZapod Why do not use pre-trained machine learning model for image classification from Tensorflow like? :
> 
> 1. [Inception](https://www.tensorflow.org/api_docs/python/tf/keras/applications/inception_v3/InceptionV3)
> 2. [Resnet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet)
> 3. [MobileNet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet/MobileNet)

That may work for now, but will fail in near future.

**Comment by annasajkh** [comment]
yeah narrow classification model will fail to detect general transformers, we need better models
